readme.md

```diff
@@ -169,10 +167,8 @@ Comment, click, like, share, up-vote, subscribe
 [discu]: https://github.com/zer0Kerbal/SimpleHyperdrive/discussions "Discussions"
 [forum]: https://forum.kerbalspaceprogram.com/index.php?/topic/212069-*/ "SimpleHyperdrive Forum Thread"
 [issue]: https://github.com/zer0Kerbal/SimpleHyperdrive/issues "Issues"
-[markt]: https://zer0kerbal.github.io/SimpleHyperdrive/Marketing "Marketing Slicks"
 [notic]: https://zer0kerbal.github.io/SimpleHyperdrive/Notices "Notices"
 [pages]: https://zer0kerbal.github.io/SimpleHyperdrive "GitHub Pages"
-[parts]: https://zer0kerbal.github.io/SimpleHyperdrive/PartsCatalog "Parts Catalog"
```

Comment:
this file / link is valid.

Reply:
The ingame title was misleading for the ION tank and hyperdrive stabilizer

Reply:
I've readded it.